addons/mass_mailing/data/mailing_data_templates.xml

```diff
@@ -125,41 +125,39 @@
     </template>
 
     <template id="mass_mailing.mass_mailing_kpi_link_trackers" name="Marketing: mailing link trackers statistic">
-        <div class="global_layout" t-if="link_trackers">
-            <table bgcolor="#ffffff" cellspacing="0" cellpadding="0" width="650" align="center" border="0" style="width: 100%; max-width: 650px;">
-                <tr>
-                    <td style="width: 100%;">
-                        <table cellspacing="0" cellpadding="0" border="0" width="580" align="center" style="width:100%; max-width:580px;">
-                            <tr>
-                                <td align="left">
-                                    <span style="color:#282f33; font-size: 15px; font-weight: bold; line-height: 30px">
-                                        <t t-esc="'Click Rate Report on %i %s Sent' % (object.expected, mailing_type)"/>
-                                    </span>
-                                </td>
-                            </tr>
-                        </table>
-                    </td>
-                </tr>
-                <tr>
-                    <td style="margin: 0; padding:0;">
-                        <table cellspacing="0" cellpadding="0" border="0" width="580" align="center" style="width:100%; max-width:580px;border-collapse: collapse;">
-                            <tr style="color: #875a7b; font-size: 16px; font-weight: 500;">
-                                <td style="width: 70%;padding: 10px 0; text-align: center; border: 1px solid #e7e7e7;">Button Label</td>
-                                <td style="width: 30%;padding: 10px 0; text-align: center; border: 1px solid #e7e7e7;">%Click (Total)</td>
-                            </tr>
-                            <tr t-foreach="link_trackers" t-as="link_tracker" style="color: #888888; font-size: 15px; font-weight: 300;">
-                                <td style="width: 70%;padding: 10px 0; border: 1px solid #e7e7e7;">
-                                    <a t-att-href="link_tracker.absolute_url" target="_blank" style="color: #56b3b5; text-decoration: none;" t-esc="link_tracker.label or link_tracker.url"/>
-                                </td>
-                                <td style="width: 30%;padding: 10px 0; text-align: center;  border: 1px solid #e7e7e7;">
-                                    <t t-esc="int(link_tracker.count * 100 / object.sent) if object.sent else 0"/>% (<t t-esc="link_tracker.count"/>)
-                                </td>
-                            </tr>
-                        </table>
-                    </td>
-                </tr>
-            </table>
-        </div>
+        <table t-if="link_trackers" class="global_table_layout" bgcolor="#ffffff" cellspacing="0" cellpadding="0" align="center" border="0" style="width: 100%;">
```

Comment:
There is no need to rename the class.

Reply:
No but I mean before this, the div this replaces just had ``class=global_layout``. I guess maybe we don't need the style, and the class only exists for the tests?

But then can we not rewrite the tests to avoid needing a class? Since it's the first element of the template anyway.
```
kpi_click_values = body_html.xpath('//table[hasclass("global_table_layout")]//tr[contains(@style,"color: #888888")]/td[contains(@style,"width: 30%")]/text()')
```
can just become
```
kpi_click_values = body_html.xpath('/table//tr[contains(@style,"color: #888888")]/td[contains(@style,"width: 30%")]/text()')
```
I believe.

Reply:
Yes, now we do not need `class=global_layout` anymore because this is the first element of the template.

Reply:
Sure but can we not remove the ``global_table_layout`` class too and just rewrite the 2 tests that use it without ``hasclass``?

Reply:
Done.